test/ETO/ETOTerms.js

```diff
@@ -372,13 +360,30 @@ contract("ETOTerms", ([deployer, admin, investorDiscount, investorNoDiscount, ..
     });
 
     it("should reject on minimum ticket too small", async () => {
+      const [modifiedConstraints] = await deployETOTermsConstraints(ETOTermsConstraints, {
+        MIN_TICKET_SIZE_EUR_ULPS: Q18.mul(200),
+      });
       const [modifiedTerms] = await redeployTerms({
-        MIN_TICKET_EUR_ULPS: (await platformTerms.MIN_TICKET_EUR_ULPS()).sub(1),
+        MIN_TICKET_EUR_ULPS: (await modifiedConstraints.MIN_TICKET_SIZE_EUR_ULPS()).sub(1),
       });
-      await expect(modifiedTerms.requireValidTerms(platformTerms.address)).to.be.rejectedWith(
+
+      await expect(modifiedTerms.requireValidTerms(modifiedConstraints.address)).to.be.rejectedWith(
         "NF_ETO_TERMS_MIN_TICKET_EUR_ULPS",
       );
     });
+
+    it("should reject on maximum ticket too high", async () => {
+      const [modifiedConstraints] = await deployETOTermsConstraints(ETOTermsConstraints, {
+        MAX_TICKET_SIZE_EUR_ULPS: Q18.mul(5000),
+      });
+      const [modifiedTerms] = await redeployTerms({
+        MIN_TICKET_EUR_ULPS: (await modifiedConstraints.MAX_TICKET_SIZE_EUR_ULPS()).add(1),
+      });
+
+      await expect(modifiedTerms.requireValidTerms(modifiedConstraints.address)).to.be.rejectedWith(
+        "NF_ETO_TERMS_MAX_TICKET_EUR_ULPS",
+      );
+    });
```

Comment:
this does not seem to be tested
```
// only allow transferabilty if this is allowed in general
        require(!ENABLE_TRANSFERS_ON_SUCCESS || termsConstraints.CAN_SET_TRANSFERABILITY(), "NF_ETO_TERMS_ENABLE_TRANSFERS_ON_SUCCESS");
```

Reply:
It's tested in ETOTerms.js, now on line 165. It was there before though.